app/Http/Controllers/Api/ServicesController.php

```
@@ -1383,6 +1385,426 @@ public function delete_env_by_uuid(Request $request)
+            'name' => $application->name,
+            'human_name' => $application->human_name,
+            'description' => $application->description,
+            'fqdn' => $application->fqdn,
+            'image' => $application->image,
+            'exclude_from_status' => $application->exclude_from_status,
+            'is_log_drain_enabled' => $application->is_log_drain_enabled,
+            'is_gzip_enabled' => $application->is_gzip_enabled,
+            'is_stripprefix_enabled' => $application->is_stripprefix_enabled,
+            'message' => 'Application updated successfully. Restart the service to apply changes.',
+        ]);
+    }
```

PHPMD (2.15.0)

The method update_application() has a Cyclomatic Complexity of 24. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)